src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/RecognisePriorLearningData.cshtml

```diff
@@ -0,0 +1,127 @@
+﻿@model SFA.DAS.ProviderCommitments.Web.Models.PriorLearningDataViewModel
+@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@{
+    ViewData["Title"] = "Add recognition of prior learning details";
+    ViewBag.ValidationOrder = "TrainingTotalHours,DurationReducedByHours,IsDurationReducedByRpl,ReducedDuration,CostBeforeRpl,ReducedPrice";
```

Comment:
As mentioned previously, there are GA tags here

Reply:
@Pauljgraham i was given the clarity on GA. We are retiring it by April 2024, but we should continue supporting the tags till then as per PAD team's guidelines on GA. 

That being, said can you add the tags for these pls?